tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
     "target": "es5",
-    "lib": ["es2015"],
+    "lib": ["es2015", "dom"],
```

Comment:
`@types/react` depends on DOM APIs. We should either specify all types packages [explicitly](https://www.typescriptlang.org/docs/handbook/tsconfig-json.html#types-typeroots-and-types) in this tsconfig to prevent `react` from being included or add DOM APIs as it is done now.

Reply:
I'd rather be specific than include DOM APIs since this project doesn't use React or the DOM directly.

Reply:
Okey, will do it than.